lib/ui/theme.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/material.dart';
+
+class AppTheme {
+  AppTheme._();
+
+  //colors
+  static const primary = Color(0xFF0097A7);
+
+  static const labelColor = Colors.white;
+
+  //styles
+  static const labelTextStyle = TextStyle(fontSize: 12.0, color: labelColor);
```

Comment:
Can be separated into two classes - `Colors` and `TextStyles`. Then we can import this file like this - `import 'path/to/theme.dart as Theme`, which can be used like `Theme.Colors.primary` or `Theme.TextStyles.label`.

Reply:
If you want to seperate them, I would rather import them as normal or put them in seperate files agein, because I think Theme.Colors.primary is rather long.
What do you think?

Reply:
Deleted the file, because no styles in there for now

Reply:
I agree. We'll create a separate one for styles, if and when needed.